Claude_Investor.ipynb

```diff
@@ -259,7 +259,8 @@
         "    return response_text\n",
         "\n",
         "def generate_ticker_ideas(industry):\n",
-        "    system_prompt = f\"You are a financial analyst assistant. Generate a list of 5 ticker symbols for major companies in the {industry} industry, as a Python-parseable list.\"\n",
+        "    system_prompt = f\"You are a financial analyst assistant with knowledge of global and Indian markets. Generate a list of 5 ticker symbols for major companies in the {industry} industry within India, as a Python-parseable list. Focus on companies listed on Indian stock exchanges such as NSE or BSE.\"\n",
```

Comment:
But what if they are interested in a non- Indian market , Better maybe include an option for Country as a input from User

Reply:
Check Latest commit